app/models/like.rb

```diff
@@ -1,4 +1,9 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+  # , counter_cache: :likes_counter
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ ] Kindly remove commented code.

- **[OPTIONAL]** I would suggest you add an active record callback for the `update_likes_counter`